src/components/ThemeToggle/ThemeToggle.test.tsx

```diff
@@ -33,20 +33,20 @@ beforeEach(() => {
     return {
       user: testUser1,
       portalUser: testPortalUser1,
-      loading: false
+      loading: false,
     }
   })
 })
 
 describe('ThemeToggle component', () => {
   test('renders nothing if no user', async () => {
-jest.spyOn(useUserHooks, 'useUser').mockImplementation(() => {
-  return {
-    user: null,
-    portalUser: null,
-    loading: true,
-  }
-})
+    jest.spyOn(useUserHooks, 'useUser').mockImplementation(() => {
```

Comment:
more formatting weirdness in test files 🥲

Reply:
So many, I wonder how all these were missed? Maybe we got a new version of prettier that does a better job?